mantidimaging/gui/windows/main/model.py

```diff
@@ -160,3 +160,7 @@ def do_remove_stack(self, stack_uuid: uuid.UUID) -> None:
     @property
     def have_active_stacks(self) -> bool:
         return len(self.active_stacks) > 0
+
+    def add_log_to_sample(self, stack_name: str, log_file: str):
+        stack = self.get_stack_by_name(stack_name).widget()  # type: ignore
+        stack.presenter.images.log_file = loader.load_log(log_file)
```

Comment:
This should also do `log.raise_if_angle_missing(stack.presenter.images.filenames)` (or something close to this) like [here](https://github.com/mantidproject/mantidimaging/blob/master/mantidimaging/gui/windows/load_dialog/presenter.py#L242), otherwise you can load a wrong log and it will break the reconstruction as it has no checks for that itself